client/src/components/PageLayout/Header.tsx

```diff
@@ -12,6 +15,31 @@ const useStyles = makeStyles({
   cursorPointer: {
     cursor: 'pointer',
   },
+  // c.f. https://material-ui.com/components/app-bar/#app-bar-with-search-field
```

Comment:
They're usually defined in the same place.  `makeStyles` are sometimes dependent on props, which gets more difficult to grok if they're in a different file.

Reply:
@timmyichen If they might depend on props then it would be better for them to stay in the same file :sweat_smile:

Reply:
The comment was there, because it references a `theme` argument.
In my understanding, that would be a composed object.
I wasn't sure, whether it is the same `theme` as the one imported. (But I haven't tried either).

That `theme` object would contain more layout properties used to properly space objects.